src/api-client.ts

```diff
@@ -1,6 +1,7 @@
-import * as FormData from 'form-data';
+/* eslint-disable no-shadow */
```

Comment:
In the migration guide, it recommended adding these to the files first then refining the eslint rules later as time goes on.